readme.txt

```diff
@@ -6,6 +6,7 @@
 
 * Fix - Fix style overrides for new view shortcodes for Genesis theme. [ECP-316]
 * Fix - Fix style overrides for new view shortcodes for Enfold theme. [ECP-315]
+* Tweak - Update `adjustStart()` function in moment.js to allow start and end time to be the same. [TEC-3009]
```

Comment:
@sc0ttkclark the `adjustStart` function is actually from our own moment utils, not from `moment.js`. So how I had written it previously is correct.

Reply:
merging to unblock https://github.com/moderntribe/the-events-calendar/pull/2998. see https://github.com/moderntribe/tribe-common/pull/1260